lara-typescript/src/section-authoring/api/lara-api-provider.ts

```diff
@@ -88,6 +92,19 @@ export const getLaraAuthoringAPI =
     return sendToLara({url: createPageItemUrl(args.pageId), method: "POST", body});
   };
 
+  const updatePageItem: APIPageItemUpdateF = (args: {pageId: string, sectionItem: ISectionItem}) => {
+    const pageItem = args.sectionItem;
+    const pageItemData = pageItem.data;
+    const translatedData: any = {};
+    Object.keys(pageItemData).forEach((key: string) => {
```

Comment:
OK, I was thinking that this entire itterator block that creates the new object would be the function extracted to a method we can reuse. Currently `cameToSnakeCase` just changes a string, but if we could have it rename all the keys in an object, that would be great.